arkouda/random/_generator.py

```diff
@@ -212,6 +214,47 @@ def standard_normal(self, size=None):
             return self._np_generator.standard_normal()
         return standard_normal(size=size, seed=self._seed)
 
+    def permutation(self, x):
+        """
+        Randomly permute a sequence, or return a permuted range.
+
+        Parameters
+        ----------
+        x: int or pdarray
+            If x is an integer, randomly permute ak.arange(x). If x is an array,
+            make a copy and shuffle the elements randomly.
+
+        Returns
+        -------
+        pdarray
+            pdarray of permuted elements
+        """
+        if _val_isinstance_of_union(x, int_scalars):
+            is_domain_perm = True
+            dtype = to_numpy_dtype(akint64)
+            size = x
+        else:
```

Comment:
yah that's a good idea!

Reply:
Added!
```python
In [5]: rng.permutation(10)
Out[5]: array([2 4 6 8 7 1 5 9 3 0])

In [6]: rng.permutation(3.0)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[6], line 1
----> 1 rng.permutation(3.0)

File ~/proj/arkouda/arkouda/random/_generator.py:241, in Generator.permutation(self, x)
    239     size = x.size
    240 else:
--> 241     raise TypeError("permtation only accepts a pdarray or int scalar.")
    243 # we have to use the int version since we permute the domain
    244 name = self._name_dict[to_numpy_dtype(akint64)]

TypeError: permtation only accepts a pdarray or int scalar.
```